### ISSUE TITLE
Collecting artifacts from the run_plugin script

### ISSUE THREAD

**Issue by brollb** [webgme/webgme#1313]
When running plugins from the cli, it would be nice to be able to specify a location for the result artifacts so artifacts can be retrieved (w/o hunting through the blob)

**Comment by pmeijer** [comment]
Add option:
```
.option('-w, --writeBlobFilesDir [string]', 'If defined will also write blob-files to %cwd%/%writeBlobFiles%')
```

Example:
```
node ./src/bin/run_plugin.js AddOnGenerator MyProject -w plugin-blobs
```

From the console:
```
warn: [gme:bin:runplugin] writeBlobFilesDir given, will also write blobs to c:\GIT\webgme\plugin-blobs
...
info: [gme:bin:runplugin] Wrote file to c:\GIT\webgme\plugin-blobs\NewAddOn.js
```

**Comment by pmeijer** [comment]
Closed by #1316
